forc-pkg/src/manifest.rs

```diff
@@ -1153,6 +1212,18 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_error_duplicate_deps_definition() {
+        assert_eq!(
+            "failed to parse manifest: duplicate 'foo' for key `dependencies` at line 1 column 1."
+                .to_owned(),
+            PackageManifest::from_dir("./tests/invalid")
+                .unwrap_err()
+                .root_cause()
+                .to_string()
+        );
+    }
+
```

Comment:
nit: consider using `unwrap` + [should_panic](https://doc.rust-lang.org/book/ch11-01-writing-tests.html#checking-for-panics-with-should_panic) to assert error messages

Reply:
That is pretty cool, I was not aware of should_panic. I'll use it.